src/app/Game.js

```diff
@@ -0,0 +1,98 @@
+import { Question } from './Question.js';
+import { Timer } from './Timer.js';
+import { Stats } from './Stats.js';
+
+export class Game {
+    constructor() {    
+        this.questions = [
+            {"name":"Banana bread","imgSrc":"/static/assets/img/banana-bread.jpg","apiTitle":"banana+bread"},
+            {"name":"Spaghetti carbonara","imgSrc":"/static/assets/img/carbonara.jpg","apiTitle":"spaghetti+carbonara"},
+            {"name":"Cheesecake","imgSrc":"/static/assets/img/cheesecake.jpg","apiTitle":"cheesecake"},
+            {"name":"Chicken pad thai","imgSrc":"/static/assets/img/chicken-pad-thai.jpg","apiTitle":"chicken+pad+thai"},
+            {"name":"Chocolate brownie","imgSrc":"/static/assets/img/chocolate-brownie.jpg","apiTitle":"chocolate+brownie"},
+            {"name":"Hamburger","imgSrc":"/static/assets/img/hamburger.jpg","apiTitle":"Hamburger"},
+            {"name":"Spaghetti aglio et olio","imgSrc":"/static/assets/img/spaghetti-aglio-olio.jpg","apiTitle":"spaghetti+aglio+olio"},
+            {"name":"Spaghetti bolognese","imgSrc":"/static/assets/img/spaghetti-bolognese.jpg","apiTitle":"spaghetti+bolognese"},
+            {"name":"Strawberry shake","imgSrc":"/static/assets/img/strawberry-shake.jpg","apiTitle":"strawberry+shake"},
+            {"name":"Vanilla ice cream","imgSrc":"/static/assets/img/vanilla-ice-cream.jpg","apiTitle":"vanilla+ice+cream"},
+        ]
+        this.startBtn = document.querySelector(".start")
+        this.instructionSection = document.querySelector(".instruction")
+        this.gameSection = document.querySelector(".game")
+        this.imgInput = document.querySelector(".dish-img")
+        this.dishNameInput = document.querySelector(".dish-name")
+        this.checkBtn = document.querySelector(".submit")
+        this.answerInput = document.querySelector(".answer")
+        this.correctAnswer = document.querySelector(".correct-answer")
+        this.gameOverBlock = document.querySelector(".game-over")
+        this.finalScoreInput = document.querySelector(".game-over .score span")
+
+        this.stats = new Stats(0)
+        
+        document.addEventListener('DOMContentLoaded', () => {
+            this.startBtn.addEventListener("click", this.startRound.bind(this), {once: true})
+            this.checkBtn.addEventListener("click", (e) => e.preventDefault())
```

Comment:
I will copy and paste previous comment: "Radek said that we should avoid one letter variables, even when they are obvious "